src/utils/gsuite/sheets.ts

```diff
@@ -0,0 +1,86 @@
+import { Applicant } from '../../types/applicant';
+
+import { google } from 'googleapis';
+
+// this is path to json file with the service account keys
+// these keys can be generated by creating a service account from the google developers console
+// change the path to that of the file that you generate to access the sheets api
+// note: if the sheets is not open to access by url, then need to make the service account email an editor
```

Comment:
Can you please go back and start turning these into JSDoc formatted comments? https://jsdoc.app/